### PR TITLE
Meiner bericht

### PR DESCRIPTION
**Pull request by kemechial**
Please review this, @casey-collab...:)

### CODE REVIEW

File: german/oktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Sie besuchen Wiesn beim Oktoberfest. Sie sehen ein sch�nes Maedchen in einem gr�nen Dirndl.
```
Comment (casey-collab) [inline german/oktoberfest.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline german/oktoberfest.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Pull request merged**